app/controllers/api/v1/items/revenue_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::Items::RevenueController < ApplicationController
+  respond_to :json
+
+  def index
+    @items = Item.most_revenue(params[:quantity])
```

Comment:
@pwentz Since we're not using any views, I think we could use local variables instead of instance variables throughout.